File: app/src/main/java/com/goals/ted/goals/Activities/CreateGoalActivity.java
package com.goals.ted.goals.Activities;

import android.net.Uri;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.Toolbar;
import android.view.Menu;

import com.goals.ted.goals.Fragments.CreateGoalFragment;
import com.goals.ted.goals.R;

public class CreateGoalActivity extends AppCompatActivity implements CreateGoalFragment.OnFragmentInteractionListener{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_create_goal);
        Toolbar toolbar = (Toolbar) findViewById(R.id.app_bar);
        setSupportActionBar(toolbar);
        ActionBar ab = getSupportActionBar();
        ab.setDisplayHomeAsUpEnabled(true);
        ab.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp);
        getSupportFragmentManager().beginTransaction().replace(R.id.container, new CreateGoalFragment())
                .commit();

    }

    @Override
    public void onFragmentInteraction(Uri uri) {

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.create_gest_menu, menu);
        return true;
    }
}

File: app/src/main/java/com/goals/ted/goals/Activities/CreateGoalActivity.kt
package com.goals.ted.goals.Activities

import android.net.Uri
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.Toolbar

import com.goals.ted.goals.Fragments.CreateGoalFragment
import com.goals.ted.goals.R
import kotlinx.android.synthetic.main.activity_create_goal.*

class CreateGoalActivity : AppCompatActivity(), CreateGoalFragment.OnFragmentInteractionListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_create_goal)
        setSupportActionBar(app_bar as Toolbar?)
        val ab = supportActionBar
        ab?.setDisplayHomeAsUpEnabled(true)
        ab?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
        supportFragmentManager.beginTransaction().replace(R.id.container, CreateGoalFragment())
                .commit()
    }

    override fun onFragmentInteraction(uri: Uri) {

    }


}
